### DISCUSSION TITLE
Can you find a way to watch the world cup for free at school?

### DISCUSSION THREAD

**Discussion by Terrinn** [3kh0/ext-remover#284] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Can you find a way to watch the world cup for free at school?

**Comment by Titanium-Network-Hack3r** [comment]
You make a free account on peacock and then you break your screen and watch it at home ;) also this is very off topic!

**Reply by Aka-but-nice** [reply]
True

**Comment by 3kh0** [comment]
Convince the teacher that the world cup is more important than school, if you have a nice teacher, they might put it one, we did that with our teacher :)

**Reply by Aka-but-nice** [reply]
IT WORKED?

**Comment by rwg100** [comment, marked as answer]
you cheer for costa rica and then instantly die when you find out spain dominated them 7-0
